contracts/colonyNetwork/ColonyNetworkDeployer.sol

```diff
@@ -93,6 +94,43 @@ contract ColonyNetworkDeployer is ColonyNetworkStorage {
     return colonyAddress;
   }
 
+  function createColonyForFrontend(
+    address _tokenAddress,
+    string memory _name,
+    string memory _symbol,
+    uint8 _decimals,
```

Comment:
IMO we should drop this argument, I expect users choose something other than 18 extremely rarely.

Reply:
That's a totally fine opinion to hold, but this isn't the place to make that decision (which needs to start at the top with product, and work down from there)